src/pages/MiniGames/MultipleChoiceQuiz/data/quiz-data.json

```diff
@@ -0,0 +1,147 @@
+[
+  {
+    "question": "What%20was%20the%20name%20of%20the%20hero%20in%20the%2080s%20animated%20video%20game%20%27Dragon%27s%20Lair%27%3F",
```

Comment:
it's odd that the strings need to be stored this way. What happens if you store the spaces normally?